magento/magento2
# Add back extension point for adding some html to the category page

### Description (*)
Moved check from the catalog to the swatches module in order to add back the extension point.

### Fixed Issues (if relevant)
1. Fixes magento/magento2#29286

### Manual testing scenarios (*)

1. See https://github.com/magento/magento2/pull/24728 and magento/magento2#29286


### Contribution checklist (*)
